### ISSUE TITLE
under which license is parse4j available

### ISSUE THREAD

**Issue by imod** [thiagolocatelli/parse4j#23]
In your `pom.xml` I see `The Apache Software License, Version 2.0` is referenced, but in your README you have a _TODO_ - so what is the official statement?

**Comment by sidiabale** [comment]
Indeed, can you clarify the license? It is Apache 2.0 or something else? I'm working on a port to Codename One and I would like to have the proper license.

**Comment by thiagolocatelli** [comment]
feel free to do whatever you want!! I just put the code out there so anyone
can do anything with it.

On Thu, Mar 5, 2015 at 2:57 PM, sidiabale <notifications@github.com> wrote:

> Indeed, can you clarify the license? It is Apache 2.0 or something else?
> I'm working on a port to Codename One and I would like to have the proper
> license.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/thiagolocatelli/parse4j/issues/23#issuecomment-77439312>
> .
>



-- 
"thiago:locatelli$gmail:com".replace(':','.').replace('$','@')

**Comment by sidiabale** [comment]
Ok. Thanks!
